lumicks/pylake/channel.py

```diff
@@ -131,6 +131,32 @@ def downsampled_over(self, range_list, reduce=np.mean, where='center'):
 
         return Slice(TimeSeries(d, t), self.labels)
 
+    def downsampled_to(self, frequency, reduce=np.mean, where='center'):
+        """Return a copy of this slice which is downsampled by `factor`
+
+        Parameters
+        ----------
+        frequency : int
+            The desired downsampled frequency downsampled (per second)
```

Comment:
`The desired downsampled frequency (Hz)`?

Reply:
oops, copy/paste from `downsampled_by` for doc consistency but forgot to change that line